services/galley/test/integration/API.hs

```diff
@@ -1776,8 +1777,8 @@ leaveConnectConversation = do
   alice <- randomUser
   bob <- randomUser
   bdy <- postConnectConv alice bob "alice" "ni" Nothing <!! const 201 === statusCode
-  let c = maybe (error "invalid connect conversation") (qUnqualified . cnvQualifiedId) (responseJsonUnsafe bdy)
-  deleteMember alice alice c !!! const 403 === statusCode
+  let c = fromMaybe (error "invalid connect conversation") (qUnqualified . cnvQualifiedId <$> responseJsonUnsafe bdy)
```

Comment:
@akshaymankar , any reason why `fromMaybe` is better here?

Reply:
Not sure why or even if I did this!